plugins/onex/skills/release/prompt.md

```
@@ -0,0 +1,1097 @@
+    "PUBLISHED",
+    "DONE",
+]
+```
+
+---
+
+## Phase 0: Scan + Validate
+
+### Step 0.1: Pre-flight Validation
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)